xml/admin_operating_services.xml

```diff
@@ -10,245 +10,204 @@
-ceph-radosgw@.service
-ceph-rbd-mirror@.service
-</screen>
+<screen>ceph-<replaceable>FSID</replaceable>@<replaceable>SERVICE_TYPE</replaceable>.<replaceable>ID</replaceable>.service</screen>
    <para>
-    To use these commands, you first need to identify the name of the service
-    you want to operate. See
-    <xref linkend="ceph-operating-services-finding-names"/> to learn more about
-    services identification.
+    For example:
    </para>
+<screen>ceph-b4b30c6e-9681-11ea-ac39-525400d7702d@mon.doc-ses-min1.service</screen>
+<screen>ceph-b4b30c6e-9681-11ea-ac39-525400d7702d@rgw.myrealm.myzone.doc-ses-min1.kwwazo.service</screen>
+   <variablelist>
+    <varlistentry>
+     <term>FSID</term>
+     <listitem>
+      <para>
+       Unique ID of the &ceph; cluster. You can find it in
+       <filename>/etc/ceph/ceph.conf</filename>.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>SERVICE_TYPE</term>
+     <listitem>
+      <para>
+       Type of the service, for example <literal>osd</literal>,
+       <literal>mon</literal>, or <literal>rgw</literal>.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>ID</term>
+     <listitem>
+      <para>
+       Identification string of the service. For OSDs, it is the ID number of
+       the service. For other services, it can be either a host name of the
+       node, or additional strings relevant for the service type.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
    <para>
-    To start, stop or restart the <literal>osd.1</literal> service, run:
+    The following commands will help you list the unit files specific to the
+    node where you run them.
    </para>
-<screen>
-&prompt.root;systemctl start ceph-osd@1.service
-&prompt.root;systemctl stop ceph-osd@1.service
-&prompt.root;systemctl restart ceph-osd@1.service
-</screen>
    <para>
-    Commands for the other service types are analogous.
-   </para>
-  </sect2>
-
-  <sect2 xml:id="ceph-operating-services-finding-names">
-   <title>Identifying Individual Services</title>
-   <para>
-    You can find out the names/numbers of a particular type of service in
-    several ways. The following commands provide results for
-    <literal>ceph*</literal> services. You can run them on any node of the
-    &ceph; cluster.
-   </para>
-   <para>
-    To list all (even inactive) services of type <literal>ceph*</literal>, run:
+    To list all (even inactive) &ceph; services and targets on a node, run:
    </para>
 <screen>
-&prompt.root;systemctl list-units --all --type=service ceph*
+&prompt.root;systemctl list-units --all --type=service,target ceph*
 </screen>
    <para>
     To list only the inactive services, run:
    </para>
 <screen>
 &prompt.root;systemctl list-units --all --state=inactive --type=service ceph*
 </screen>
-   <para>
-    You can also use <command>salt</command> to query services across multiple
-    nodes:
-   </para>
+   <tip>
```

Comment:
It feels that this belongs more in the "Troubleshooting" section. We have a more straightforward way to retrieve all daemons (`ceph orch ps`). We might mislead people by stating this here

Reply:
ok, commented out for now